docs/csharp/whats-new/index.md

```diff
@@ -9,7 +9,7 @@ ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
 
 This page provides a roadmap of new features in each major release of
 the C# language. The following links provide detailed information on the
-major features added in each release.
+major features added in each release. This section provides information on new feature that have been released, either in a general release, or in a public preview. Detailed language feature status, including features considered for upcoming releases can be found [on the roslyn repository](https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md) on GitHub.
```

Comment:
on new feature or new features?

dotnet/roslyn repository? we usually put org/reponame unless you think it's not needed for roslyn